### ISSUE TITLE
Feature request: Add chart with total stars collected (from all participants)

### ISSUE THREAD

**Issue by keriati** [jeroenheijmans/advent-of-code-charts#110]
Thank you for this awesome extension!

One interesting chart would be to see how the total amount of stars grew that all people collected on the board.

**Comment by jeroenheijmans** [comment]
Thanks for your kind words, and the suggestion!

Can you elaborate what you mean by "from all participants", and how it relates to this existing graph?

![image](https://github.com/user-attachments/assets/fd41dd16-63b2-40c9-b794-dc0b88ff17af)

Are you suggesting some kind of "cumulative total" single line with the data from that graph? If so, out of curiosity: what would that graph 'represent'? Or if it's something else, please let me know what you'd like to see/do with the graph.

Happy puzzling!

**Comment by keriati** [comment]
Yes, a cumulative total. Essentially, the idea is to display something like: on this leaderboard, all members together have collected a total of 123 stars so far. Additionally, it would be great to have this represented as a graph to visualize how the total changes over time. It might seem like a small thing, but for our company leaderboard, it could be a fascinating way to track overall activity. Plus, it would allow us to compare this year’s activity with that of the previous year.

**Comment by jeroenheijmans** [comment]
Cheers, that's clear now!

Knowing that the rationale is to see "participation per day" (and probably compared to previous years) is helpful in creating this feature.

In fact, it feels at its most useful if the comparison with previous years is there. But that will require me to think about smarter caching because I don't want to spam AoC's API and each year is a separate call.

Thanks again for the suggestion!
